### ISSUE TITLE
Modelar habilidades

### ISSUE THREAD

**Issue by leonardoaj** [midianinja/AppNinja#49]
O que são habilidades? Quais são os atributos? Somente texto descritivo mesmo?

**Comment by BrancaSchulz** [comment]
As habilidades são referentes a capacidade de uma pessoa fazer alguma coisa, pode ser desde uma habilidade social, profissional à cognitiva. 

A sugestão é iniciarmos por uma lista de seleção de habilidades profissionais. Segue uma proposta de lista a partir dos insumos extraídos do Cadastro - Somos Resistência. Seja NINJA! 


Advogado(a)
Agriculator(a)
Agricultor(a)
Ambientalista
Analista
Analista de Dados
Animador
Antropólogo(a)
Arqueólogo(a)
Arquiteto(a)
Artesã
Articulador(a)  cultural
Artista
Artista Plástico
Artista Visual
Assessor(a) Comunicação
Assistente de edição
Assistente social
Ativista
Atleta
Ator
Bailarino(a)
Biólogo(a)
Cabelereiro(a)
Cartunista
Cenógrafo
Chargista
Chef de Cozinha
Ciclista
Cientista
Cientista da Informação
Cientista de Dados
Cientista Político
Cineasta
Cinegrafista
Comentarista de Futebol
Compositor
Comunicador(a)
Contador(a)
Costureira
Defensor(a) Pública
Dentista
Desenho de som
Design UX
Designer
Designer de Interiores
Designer em Moda
Designer Gráfico
Direitor Audiovisual
Diretor de arte
Economista
Editor
Editor de vídeo
Educador(a)
Enfermeiro(a)
Engenheiro(a)
Engenheiro(a) de Som
Escritor(a)
Estilista
Estudante
Figurinista
Filósofo(a)
Físico
Fisioterapeuta
Fonodiólogo
Fotógrafo Amador
Fotógrafo Documental
Fotógrafo(a)
Fotojornalista
Gamer
Geógrafo(a)
Gestão em políticas públicas
Hacker
Historiador(a)
Indigenista
Influencer Digital
Interprete
Interprete de Libras
Jornalista
Jornalista de moda e beleza
Legenda
Logger/Acervo
Making Off
Maquiador
Maquiagem
Marketing
Marketing Digital
Médico(a)
Museólogo
Musicista
Músico
Musicoterapeuta
Nutricionista
Operador de Camera
Performer
Pesquisador(a)
Pianista
Poeta
Pós produção
Pós-produção de Som
Produtor
Produtor Musical
Produtor(a) Cultural
Professor de audiovisual
Professor de Educação Física
Professor Universitário
Professor(a)
Professora Rede Pública
Programador de software
Psicólogo(a)
Psicopedagogo(a)
Publicitário(a)
Radialista
Rapper
Redator
Reiki
Relações Internacionais
Relações Públicas
Reporter
Roteirista
Segurança da informação
Segurança dos Alimentos
SEO
Serviço Social
Servidor Público
Sindicalista
Sistema tributário
Social Media
Sociólogo
Sound-Designer
Surfista
Taróloga
Tatuador
Técnico Áudio / Som direto
Técnico em Informática
Terapeuta Holística
Tradutor
Trilha sonora
Turismologo(a)
Urbanista
Videomaker
Violoncelista
VJ
Yogi
Outros

Avalio que é necessário colocar o campo de outros e tb permitir que o usuário selecione no máximo 3 habilidades.

**Comment by leonardoaj** [comment]
sem problemas. a modelagem atualmente proposta contempla esta forma de organização dos dados. entretanto, fica difícil limitar a 3 habilidades somente. acredito que tal limitação (se for de fato imposta) deva ser implementada no front-end.

**Comment by BrancaSchulz** [comment]
@leonardoaj Ok não ter limitação 
Apenas sugestão, temos histórico de pessoas que selecionam o máximo de possibilidades e há dificuldade de entender quais os pontos priortários de diálogo com elas.

**Comment by leonardoaj** [comment]
sim, também acho válido haver um limite, faz com que o usuário se defina melhor. mas do ponto de vista do sistema, que eventualmente receberia simplesmente uma lista de habilidades, fica difícil decidir quais poderiam ser escolhidas e quais descartadas. Caso essa decisão seja "imposta" pelo front, por exemplo, desabilitando os campos após a escolha de N habilidades, isso faria com que o próprio usuário definisse o que é mais relevante para si próprio.
